### DISCUSSION TITLE
Does the Google Colab notebok install a1111 on my google drive?

### DISCUSSION THREAD

**Discussion by MortazaG** [TheLastBen/fast-stable-diffusion#2533] [General]
Hi, I'm new to using a1111 and google colab. I've been using the notebook from the stable-diffusion-art website, but I am not sure if it installs a1111 on my drive or not.

How does TheLastBen colab notebook work, is a1111 installed on my drive? The reason I am asking is because I would like a notebook that actually installs a1111 on my drive so that I can launch the web-ui faster.

**Comment by TheLastBen** [comment]
yes, the notebook installs all the files and the extensions and models in your gdrive, so you can resume everything anytime you want, even if you update the notebook, the files will stay

**Reply by MortazaG** [reply]
Nice, thank you for the reply! I have now been trying out the notebook and have some questions that I would appreciate if you or someone else could answer.

1. Why is it necessary to run all the cells everytime, even when all the files are already on my google drive? I understand the first cell, having to connect to Google Drive, but is there someway of skipping the rest of the cells and go directly towards the Start Stable Diffusion cell? Currently it takes around 5-7 minutes to run through them all and start the web-ui, with the T4 GPU and extra RAM on Colab Pro. Is this the normal time it takes for most people?

2. Is this line in the Requirements cell correct:
   `if not os.path.exists('/content/gdrive/'+mainpth+'/sd/stablediffusiond'): #restore later`
   In my drive the path is '/sd/stablediffusion', without the d at the end, is this a typo in the code?

**Reply by TheLastBen** [reply]
Which cell takes 7 minutes ?

The d at the end is an intentional typo to force an update

**Reply by MortazaG** [reply]
> Which cell takes 7 minutes ?
> 
> The d at the end is an intentional typo to force an update

Oh okey, I understand.

All of the cells together take up to 7 minutes to go through, even with all the models and everything I need previously downloaded and saved on my gdrive. Is it normal for it to take so long or am I doing something wrong?

**Reply by TheLastBen** [reply]
it's mostly the last cell, loading the models from gdrive takes longer than normal
